ChromeExtension/manifest.json

```diff
@@ -1,43 +1,60 @@
 {
-    "manifest_version": 2,
+    "manifest_version": 3,
 
     "name": "Geniuslink | Intelligent Link Creator",
     "description": "Easily build geni.us links with the click of a button!",
-    "version": "1.0.5",
+    "version": "1.0.6",
 
-    "browser_action": {
+    "action": {
         "default_icon": "images/icon.png",
         "default_popup": "apikeys.html"
     },
+
     "icons": {
         "16": "images/icon.png",
         "48": "images/icon48.png",
         "128": "images/icon128.png"
     },
+
     "content_scripts": [{
         "matches": ["http://*/*", "https://*/*", "*://*/*"],
-        "js": ["js/openDialog.js"]
+        "js": ["js/jquery.min.js", "js/servicestack.js", "js/GeniusLinkServiceClient.js", "js/openDialog.js", "js/background.js"]
     }],
+    
     "web_accessible_resources": [
-        "css/custom.css",
-        "js/frame.js",
-        "alertDoneInside.html", "alertLoadingInside.html", "alertDoneOutside.html", "alertLoadingOutside.html", "css/vsprites.svg", "css/main.css", "js/bootstrap.min.js"
+        {
+            "resources": [
+                "css/custom.css",
+                "js/frame.js",
+                "alertDoneInside.html", "alertLoadingInside.html", "alertDoneOutside.html", "alertLoadingOutside.html", "css/vsprites.svg", "css/main.css", "js/bootstrap.min.js"        
+            ],
+            "matches": [
+              "*://*/*"
+            ]
+        }
     ],
 
     "background": {
-        "scripts": ["js/background.js", "js/jquery.min.js", "js/jquery-ui.min.js", "js/bootstrap.min.js", "js/bootbox.min.js", "js/utilities.js", "js/servicestack.js", "js/GeniusLinkServiceClient.js"],
-        "persistent": true
+        "service_worker": "js/serviceWorker.js"
```

Comment:
https://developer.chrome.com/docs/extensions/develop/migrate/to-service-workers#update-bg-field